web/modules/custom/migrate_subject_data/composer.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "drupal/migrate_subject_data",
+  "description": "Migrate subject data from taxonomy to node.",
+  "type": "drupal-module",
+  "autoload": {
+    "psr-4": {
+      "Drupal\\migrate_subject_data\\": "src/"
+    }
+  },
+  "extra": {
+    "drush": {
+      "services": {
+        "drush.services.yml": "^9"
+      }
+    }
+  }
```

Comment:
Out of curiosity: What does this do?

Reply:
"The drush.services.yml file needs to be declared in your composer.json for Drush 9 commands to work properly. Here, you can also define multiple services files to be used for different versions of Drush. In Drush 9, defining the services files explicitly in the composer.json file is not mandatory, and if the file is not defined, the default drush.services.yml file will be loaded.

However, from Drush 10 onwards, the services section in the composer.json file must exist, and it should name the services file to be used. Therefore, it is recommended to declare the services file in Drush 9 too." - https://www.axelerant.com/resources/team-blog/how-to-write-custom-drush-9-commands-for-drupal-8